rust/examples/socket/ftp_login.nasl

```diff
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: 2024 Greenbone AG
+#
+# SPDX-License-Identifier: GPL-2.0-or-later WITH x11vnc-openssl-exception
+
+display("Start");
+display("is function open_sock_tcp defined: ", defined_func("open_sock_tcp"));
+sock = open_sock_tcp(21, transport: 1);
+display("was socket created: ", !isnull(sock));
+display("fd: ", sock);
+display("is function ftp_log_in defined: ", defined_func("ftp_log_in"));
+# Login data for ftp://ftp.dlptest.com/ provided by https://dlptest.com/ftp-test/
```

Comment:
q: are we allowed to just use that in our public repository?

Reply:
It is a public test server for ftp. As we do not spam this server with automated test I don't see a problem here.